test_schemas/en/test_new_skip_condition_block.json

```diff
@@ -0,0 +1,99 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Skip group",
+    "theme": "default",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "id": "do-you-want-to-skip-group",
+                    "title": "Do you want to skip the next block?",
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "do-you-want-to-skip",
+                            "question": {
+                                "id": "do-you-want-to-skip-question",
+                                "title": "Do you want to skip?",
```

Comment:
I think we need to make the schema a bit clearer.

The question title should be like : `Do you want to skip the next question?`.

Then some guidance like `Select "Yes" to skip the next question and go straight to the summary`.